src/ConnectedMode/Install/SonarLintTargets.xml

```diff
@@ -25,6 +25,9 @@
 
     <ItemGroup Condition="$(BuildingInsideVisualStudio) == 'true' AND Exists($(_SLVSGeneratedRoslynConfigFile))">
         <EditorConfigFiles Condition="Exists($(_SLVSGeneratedRoslynConfigFile))" Include="$(_SLVSGeneratedRoslynConfigFile)" />
```

Comment:
FYI the analyzer config files don't seem to appear in the `SolutionExplorer` even if you add `<Visible>true</Visible>`.